recipes/conda-ecosystem-user-package-isolation/build.sh

```diff
@@ -0,0 +1,4 @@
+mkdir -p $PREFIX/etc/conda/activate.d
+mkdir -p $PREFIX/etc/conda/deactivate.d
+cp $RECIPE_DIR/activate-user-package-isolation.sh $PREFIX/etc/conda/activate.d/user-package-isolation.sh
```

Comment:
This should include `.fish` and `.csh` versions of the activation and deactivation scripts to allow it to work regardless of the user's shell.

Reply:
Ok.

Unfortunately I have exactly zero experience with these shells (csh I know by name only, fish I have never even heard of before), so I don't really know what it takes to translate these scripts to csh or fish. Any ideas anyone?

Also, I guess instead of having three versions, it might be preferrable to have one version that works on all shells if such a thing is possible. If such a thing is possible, what would the correct suffix for such a script be?

Reply:
It would be nice if this included Windows stuff, too, but we should not impose that on @fgp.  Instead, I think we should merge this as-is and create issues that some future contributor may address.